### ISSUE TITLE
release: v20.2.0

### ISSUE THREAD

**Issue by arulajmani** [cockroachdb/cockroach#56435]
Candidate RC Issue: #56145 
 
Working Tags:
- Build ID: `v20.2.0-rc.3-2-g150c5918fb`
- `staging-*` tag: `WIP EDIT ME WIP.`
 
[Release process checklist](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process)

Release date: `Monday, November 9 2020`
- [ ] [Check cluster status](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/328859690/Release+Qualification#Monitoring-the-Qualification-Clusters)
  - [ ] Destroy clusters
- [x] [Publish Cockroach Release](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-11.PublishTheRelease)
- [ ] Acked security@ and release-engineering-team@ on the generated AWS S3 bucket write alert to confirm these writes were part of a planned release (See [Publish Cockroach Release](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-11.PublishTheRelease) for details).
- [x] [Check binaries](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-14.CheckBinaries)
- [x] [Check docker image](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-15.Checkthedockerimage)
- [ ] If applicable, [update the map that roachtests use to map a version to a previous version, to reference the newly tagged version](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-12.Updateroachtestmapthattracksthesequenceofversionsacrossmajorreleasefamilies)
- [ ] [Remove staging branch / tags from repository](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Removingstagingbranch/tags)
 
- For production or stable releases in the latest major release series (in August 2020, this is the v20.1 series)
  - [x] Update the [Homebrew Formula](https://github.com/cockroachdb/homebrew-tap/blob/master/Formula/cockroach.rb)
  - [x] Update the image tag in our [Orchestrator configurations](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-Orchestratorconfigurations)
 
- For production or stable releases
  - [x] Announce version to the [registration cluster](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-AnnounceVersionToRegCluster)
 
- [ ] Update docs
- [ ] External communications for release
 
Cleanup:
 
- [ ] Remove staging branch / tags from repository
- [ ] Destroy roachprod clusters

**Comment by rmloveland** [comment]
Hi @arulajmani 

There may be some additional work needed during the homebrew update to ensure that the [spatial support](https://www.cockroachlabs.com/docs/v20.2/spatial-features.html) works in v20.2.0 for brew users.

There is additional fiddliness re: dynamic C lib locations which is documented here:

https://www.cockroachlabs.com/docs/v20.2/install-cockroachdb-spatial.html

tl;dr: `libgeos.dylib` and `libgeos_c.dylib` need to be in one of the following locations for `cockroach` to see them:
- `/usr/local/lib/cockroach/`
- a `lib/` subdirectory in the same directory as `cockroach`

Let me know if this is too much to add to the release steps, and a separate issue is needed instead

**Comment by otan** [comment]
(i'm hopeful the above "just works" -- you can test this by when updating brew, you can run `brew reinstall ./Formula/cockroach.rb` and verify `SELECT ST_IsValid(ST_MakePoint(1 1))` works.

**Comment by arulajmani** [comment]
I tried the steps @otan mentioned above, and I get the following error:

```
root@127.0.0.1:50226/movr> SELECT ST_IsValid(ST_MakePoint(1, 1));
ERROR: st_isvalid(): geos: error during GEOS init: geos: cannot load GEOS from dir "/usr/local/lib/cockroach": geos error: dlopen(/usr/local/lib/cockroach/libgeos.dylib, 1): image not found
```

@rmloveland could you ask about this on the #release slack channel please?
